graphlearn/examples/tf/ego_rgcn/train_supervised.py

```diff
@@ -183,37 +117,25 @@ def main(unused_argv):
                   FLAGS.num_blocks,
                   agg_type=FLAGS.agg_type,
                   dropout=FLAGS.drop_out)
+  # prepare train dataset
+  train_data = EgoRGCNDataLoader(g, gl.Mask.TRAIN, FLAGS.sampler, FLAGS.train_batch_size,
+                                 node_type='i', nbrs_num=nbrs_num, num_relations=FLAGS.num_relations)
+  train_embedding = model.forward(train_data.as_list(), nbrs_num)
+  loss = supervised_loss(train_embedding, train_data['seed'].labels)
```

Comment:
The user cannot know 'seed', it is inside 'train_data`, so maybe just use API like `seed().labels` ?

Reply:
I have changed back to `train/test/val` based on the `mask` parameter (keep the previous behaviour) and expose some helpers `train_labels`, `test_labels`, `val_labels` for accessing.